packages/commonwealth/client/scripts/controllers/app/login.ts

```diff
@@ -397,6 +397,8 @@ function getProfileMetadata({ provider, userInfo }): {
     return { username: userInfo.name, avatarUrl: userInfo.profile };
   } else if (provider === 'google') {
     return { username: userInfo.name, avatarUrl: userInfo.picture };
+  } else if (provider === 'apple') {
```

Comment:
Is this line required given that an empty object is the default return for the function?